kafka-connect-runtime/src/test/java/io/tabular/iceberg/connect/rest/RESTIntegrationCdcTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.tabular.iceberg.connect.rest;
+
+import io.tabular.iceberg.connect.AbstractIntegrationCdcTest;
+import io.tabular.iceberg.connect.TestConstants;
+
+public class RESTIntegrationCdcTest extends AbstractIntegrationCdcTest {
```

Comment:
I feel we can add a catalog type parameter to the tests so we don't need these subclasses.

Reply:
Another alternative is to combine all of the test classes into one abstract test class so you will only need to create one subclass per catalog. This is similar to Iceberg's [CatalogTest](https://github.com/apache/iceberg/blob/768e516e0b138a6e996fe38fe5be4a0f624862db/core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java#L69). I recently made a change so the tests shouldn't need to be in separate classes anymore.